www/src/data/sidebars/doc-links.yaml

```diff
@@ -169,6 +169,19 @@
           link: /docs/post-css/
         - title: TailwindCSS
           link: /docs/tailwind-css/
+    - title: MDX
+      link: /docs/mdx/
+      items:
```

Comment:
the links might need trailing / . Not sure!

Reply:
oh, also the guide categories aren't part of the URL currently. So take out the `/mdx/` for now.

Reply:
Oh, now that I think about it, taking out the `/mdx/` from those paths would mean retitling some of these docs so the URLs are more readable and memorable. For example, `/docs/getting-started/` is a bit vague and actually might already be taken, so it'd have to be `/docs/getting-started-with-mdx/` or something like that.

@fk, did we have a reason for not including guide categories in the pathnames?

Reply:
Good question … I think we didn't even get this far. 
One reason to explicitly not do so could have been not to have to write a lot of redirects (from old to new URL/path).

Reply:
I'm cool using either url structure, just let me know which is preferred. We'd be able to change the filenames slightly to ensure they won't conflict:

- /docs/getting-started-with-mdx
- /docs/programmatically-creating-pages-with-mdx
- /docs/writing-mdx-pages

Reply:
Sorry, got sidetracked.
Thinking out loud, with a little more time on my hands:

* looking at https://github.com/gatsbyjs/gatsby/tree/master/docs/docs I can follow @DSchau's comment — I don't see anything wrong with a little more organization there
* URL schemas are another topic, one that I am guilty of overthinking 😅 … def. can see some advantages in "the flatter the better"
* related-unrelated: do our docs categories titles translate 1:1 to URL pathnames? 
![image](https://user-images.githubusercontent.com/21834/52007890-c1d06100-24cf-11e9-9594-696c54716920.png)
* if we would include docs categories in the URL, we'd  `/docs/guides/mdx/` 😉 

… going in circles now 😆 

I think right now it's not hurting to go with `/mdx/...`.
I would personally like to see some more structure in both the filesystem and the URLs.
If we decide to do so, we need to add some redirects either way, no matter if we keep things flat or not now.